### PR TITLE
vanilla skins only list now only shows vanilla skins

### PR DESCRIPTION
**Pull request by fokkonaut**

### CODE REVIEW

**Comment by fokkonaut** [comment]
what happened to travis bot?

File: src/game/client/components/skins.cpp [hunk 1]
```diff
@@ -172,7 +174,14 @@ int CSkins::Num()
 
 const CSkins::CSkin *CSkins::Get(int Index)
 {
-	return &m_aSkins[max(0, Index%m_aSkins.size())];
+	if(Index < 0)
+	{
+		Index = Find("default");
+
+		if (Index < 0)
```
Comment (Learath2) [inline src/game/client/components/skins.cpp]: Keep the max and you don't need this

Comment (heinrich5991) [reply src/game/client/components/skins.cpp]: I actually like the non-max version better, the control flow is clearer to me.

Comment (Learath2) [reply src/game/client/components/skins.cpp]: I'd probably replace the entire thing with `Index = Index < 0 ? Find("default") : Index;` and keep the max.

**Comment by Learath2** [comment]
Thanks
bors r+

**Pull request merged**